### PR TITLE
US004 - Add branding to the product

### PR DESCRIPTION
**Pull request by AZZohaib**
image

### CODE REVIEW

File: views/components/footer.njk [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<footer class="d-flex fixed-bottom flex-wrap justify-content-between align-items-center py-3 border-top">
+    <p class="col-md-4 mb-0">© 2024 Kainos</p>
+    
+    <div class="nav-logo-container">
+        <img src="../assets/index/kainos-logo.png" alt="transparent kainos logo for the navbar" class="img-fluid">
+    </div>
+    
+    <ul class="nav col-md-4 justify-content-end gap-4">
+        <li class="nav-item">123 Kainoos Road</li>
```
Comment (Leebee1392) [inline views/components/footer.njk:9]: I love this, but please put the proper Kainos address and company name

**Review by Leebee1392 — CHANGES_REQUESTED** [review]

**Review by jemorak — APPROVED** [review]

File: views/components/footer.njk (hunk 1 above)
Comment (AZZohaib) [reply views/components/footer.njk]: Done

Comment (AZZohaib) [reply views/components/footer.njk]: [028 9057 1100](https://www.google.com/search?q=kainos&rlz=1C5GCEM_enGB1120GB1120&oq=kainos&gs_lcrp=EgZjaHJvbWUqBwgAEAAYjwIyBwgAEAAYjwIyEAgBEC4YxwEYsQMY0QMYgAQyBggCEEUYPDIGCAMQRRg8MgYIBBBFGDwyBggFEEUYPDIGCAYQRRg8MgYIBxBFGDzSAQgxNjg3ajBqN6gCCLACAQ&sourceid=chrome&ie=UTF-8#)

Comment (AZZohaib) [reply views/components/footer.njk]: image

**Review by Leebee1392 — APPROVED** [review]

**Pull request merged**